Preprocessing/Split.ecl

```diff
@@ -0,0 +1,73 @@
+/*###################################################################################
+## HPCC SYSTEMS software Copyright (C) 2020 - 2021 HPCC Systems.  All rights reserved.
+##################################################################################### */
+
+IMPORT $.^ AS MLC;
+IMPORT $ AS Preprocessing;
+IMPORT Preprocessing.Types AS Types;
+
+NumericField := MLC.Types.NumericField;
+
+/**
+ * Split input data into training and test sets based on the split ratio. It requires the data has sequential id starting with 1.
+ *
+ * @param dataToSplit: DATASET(Types.NumericField).
+ *   <p> The data to split.
+ *
+ * @param trainSize: REAL4, DEFAULT = 0.0.
```

Comment:
If these are sizes (i.e. record counts), why is this a real?  These must be fractions of the record count???  Please specify.